wandb/sdk/artifacts/artifacts_cache.py

```diff
@@ -126,6 +164,13 @@ def cleanup(self, target_size: int, remove_temp: bool = False) -> int:
 
             total_size -= stat.st_size
             bytes_reclaimed += stat.st_size
+
+        if total_size >= target_size:
```

Comment:
```suggestion
        if total_size > target_size:
```
Only warn if we couldn't hit the target (for example don't warn on `0 >= 0`)